src/ui/components/DefinitionList/index.js

```diff
@@ -0,0 +1,50 @@
+/* @flow */
+import * as React from 'react';
+import makeClassName from 'classnames';
+
+import './styles.scss';
+
+
+type DefinitionItemProps = {|
+  children: React.Node,
+  className?: string,
+  title: React.Node,
+|};
+
+export const DefinitionItem = (
```

Comment:
I think it would be nice to just call this `Definition` since that's pretty much what a `<dd>` stands for. The fact that `<dd>` requires a separate `<dt>` tag is really weird to me.